disperser/batcher/metrics.go

```diff
@@ -115,9 +122,37 @@ func NewMetrics(httpPort string, logger common.Logger) *Metrics {
 		),
 	}
 
+	finalizerMetrics := FinalizerMetrics{
+		Blobs: promauto.With(reg).NewCounterVec(
+			prometheus.CounterOpts{
+				Namespace: namespace,
+				Name:      "finalizer_num_blobs",
+				Help:      "number of blobs in each state",
+			},
+			[]string{"state"},
+		),
+		LastFinalizedBlock: promauto.With(reg).NewGauge(
```

Comment:
Why tracking this since last finalized block is a public known fact?

Reply:
It would be useful to know what `LastFinalizedBlock ` the system sees

Reply:
Ok, maybe `LastSeenFinalizedBlock` to indicate it's tracking system's progress?